docs/layouts/shortcodes/get-started/basic-app.md

```diff
@@ -0,0 +1,111 @@
+<div x-show="stack === 'php'">
+
+```yaml {location=".platform.app.yaml"}
```

Comment:
The PHP alternative to surface here is when there's an actual start command and need for a socket. Swoole as an example:

https://docs.platform.sh/languages/php/swoole.html#use

```
web:
    upstream:
        socket_family: tcp
        protocol: http
    commands:
        start: php path/to/swoole/start/command --port=$PORT
    locations:
        "/":
            allow: false
            passthru: true
```

Reply:
There won't really be any variation on the use of composer as a PM, except in the cases where we've found the Drupal/WordPress forego dependency management (and even Git) altogether. 

- [Why use Composer?](https://docs.platform.sh/guides/wordpress/composer.html)
- [Deploying without Composer](https://docs.platform.sh/guides/wordpress/vanilla.html)